src/gateways/createArchiveLoader.ts

```diff
@@ -23,7 +49,7 @@ export default (url: string) => {
     return archiveFetch<ArchiveContent>(`${url}/contents/${id}`)
       .then((response) => {
         cache.set(id, response);
-        return response;
+        return cleanArchiveResponse(response);
```

Comment:
do the call to `cleanArchiveResponse` before the result is cached on the previous line, otherwise a second call to the same url will not have the cleaned result